packages/create-plugin/templates/scenes-app/src/pages/WithDrilldown/WithDrilldown.tsx

```diff
@@ -66,7 +66,7 @@ const getDrilldownsAppScene = () => {
 
               return new SceneAppPage({
                 url: prefixRoute(`${ROUTES.WithDrilldown}`) + `/room/${roomName}/temperature`,
-                title: `${roomName} overview`,
+                title: `${decodeURI(roomName)} overview`,
```

Comment:
This change is fixing an issue that was addressed in old dead repo (see [here for context](https://github.com/grafana/scenes-app-template/pull/2))